docs/configuration/intra_node_data_path.md

```diff
@@ -183,3 +190,75 @@ On the other hand, CARET requires users to provide the following description if
 
 User have to fill callback name in `variable_passings`, `publishes`'s `callback_name`. `context_type` must be set as `callback_chain`.
 After editing, use path.verify() described in the beginning of this section to verify that it has been set correctly.
+
+### Manipulating the object
+
+CARET serves Python-based APIs to define intra-node data path.
+The following commands enable `Architecture file editing` done in the previous section to be implemented by python commands.
+
+All of the following code snippets can be executed after the `Architecture('type', 'file_path')` method loads an architecture object.
```

Comment:
```suggestion
All of the code snippets provided below can be executed after loading an architecture object using the `Architecture('type', 'file_path')` method.
```

Reply:
Fixed it in [9770137](https://github.com/tier4/CARET_doc/pull/192/commits/977013785203771f1c5ce427ab8f2e544a807b88).

The text `All of the following code snippets...` is based on the following pages.

https://tier4.github.io/CARET_doc/latest/configuration/load_and_save/#python-api

Reply:
OK.